tools/wireshark/sox.lua

```diff
@@ -141,7 +141,7 @@ local sox_cmds = {
 };
 
 function add_string(tree, field, buf, offset)
-  local str = buf(offset):stringz("utf-8")
```

Comment:
utf-8 is backward compatible with ASCII, can you explain more about it ?

Reply:
I'm think, problem was that stringz doesn't take any arguments.

I hope this a correct man page for this API:
https://wiki.wireshark.org/LuaAPI/Tvb#tvbrange:stringz.28.29